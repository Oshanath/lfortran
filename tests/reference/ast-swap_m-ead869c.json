{
    "basename": "ast-swap_m-ead869c",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../examples/swap_m.f90",
    "infile_hash": "f7ce724ae725b9a94f550a8e6932fb263d61b3e6740da3608846533e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-swap_m-ead869c.stdout",
    "stdout_hash": "037a75c231d72e6eec2a35addf985b20889b49af327bf85862f7578c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}