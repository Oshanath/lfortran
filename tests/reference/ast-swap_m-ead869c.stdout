(TranslationUnit [(Module swap_m () [] [(ImplicitNone [] ())] [(Declaration () [(SimpleAttribute AttrPrivate)] [] ()) (Declaration () [(SimpleAttribute AttrPublic)] [(swap_t [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)])) (Template swap_t [T] [(Declaration () [(SimpleAttribute AttrPrivate)] [] ()) (Declaration () [(SimpleAttribute AttrPublic)] [(swap [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)])) (DerivedType T [] (TriviaNode [] [(EndOfLine) (EndOfLine)]) [] [] [])] [(Subroutine swap_ [(x) (y)] [] () () [] [] [] [(Declaration (AttrType TypeType [] T None) [(AttrIntent InOut)] [(x [] [] () None ()) (y [] [] () None ())] ()) (Declaration (AttrType TypeType [] T None) [] [(tmp [] [] () None ())] (TriviaNode [] [(EndOfLine) (EndOfLine)]))] [(= 0 tmp x ()) (= 0 x y ()) (= 0 y tmp ())] [])])] [])])
